docs/content/en/docs/reference/changelog.md

```diff
@@ -6,7 +6,18 @@ weight: 35
 
 ## Unreleased
 
+### Added
+- Workload clusters full lifecycle API support for vSphere and Docker ([#1090](https://github.com/aws/eks-anywhere/issues/1090))
+- Single node cluster support for Bare Metal provider
+- Cilium updated to version `v1.11.10`
+- CLI high verbosity log output is automatically included in the support bundle after a CLI `cluster` command error ([#1703](https://github.com/aws/eks-anywhere/issues/1703) implemented by [#4289](https://github.com/aws/eks-anywhere/issues/1703))
+- Ability to configure rolling upgrade for Bare Metal via `maxSurge` and `maxUnavailable` parameters
+
```

Comment:
```suggestion
- New curated packages
  - Prometheus `v2.39.1`
- New versions of curated packages
  - ADOT `v0.23.0`
  - Emissary `v3.3.0`

```

Reply:
This is good, but might be nice to have `- ADOT v0.23.0 upgraded from v0.21.1`

Reply:
Sounds good, I'll update it for Emissary to, what was the previous version?